src/app/rpmostree-compose-builtin-tree.c

```diff
@@ -628,6 +628,9 @@ impl_install_tree (RpmOstreeTreeComposeContext *self,
       return TRUE;
     }
 
+  /* Print version number */
+  g_print ("RPM-OSTree Version: %s\n", PACKAGE_VERSION);
```

Comment:
I was going to suggest moving this earlier up in the function, but right, now I see: this is because of `--print-only` handling above, right? How about we switch this to `g_printerr` instead and move it at function entry? Basically, the earlier we print it, the more valuable it is since it precedes even more error paths.